src/error/models/automationError.ts

```diff
@@ -0,0 +1,11 @@
+export class AutomationError extends Error {
+    constructor(message: string) {
+      super(message);
+      this.name = "AutomationError";
```

Comment:
instead of using free text you can use `this.name = AutomationError.name` will do the same